apps/http-proxy-specific-port.yaml

```diff
@@ -1,3 +1,12 @@
+meta:
+  name: "Local HTTP Proxy demo with specific local port"
+  description: "An minimal example presenting a local HTTP proxy connecting to
```

Comment:
```suggestion
  description: "A minimal example presenting a local HTTP proxy connecting to
```

Reply:
oops... an artifact of multiple iterations over the content ;)